patch_api/api/estimates_api.py

```diff
@@ -192,6 +194,152 @@ def create_bitcoin_estimate_with_http_info(
+                                 (connection, read) timeouts.
+        :return: EstimateResponse
+                 If the method is called asynchronously,
+                 returns the request thread.
+        """
+        kwargs["_return_http_data_only"] = True
+        return self.create_ethereum_estimate_with_http_info(
+            create_ethereum_estimate_request, **kwargs
+        )  # noqa: E501
+
+    def create_ethereum_estimate_with_http_info(
+        self, create_ethereum_estimate_request, **kwargs
+    ):  # noqa: E501
+        """Create an ethereum estimate given a timestamp and gas used  # noqa: E501
+
+        Creates an ethereum estimate for the amount of CO2 to be compensated. An order in the `draft` state may be created based on the parameters, linked to the estimate.   # noqa: E501
+        This method makes a synchronous HTTP request by default. To make an
+        asynchronous HTTP request, please pass async_req=True
+        >>> thread = api.create_ethereum_estimate_with_http_info(create_ethereum_estimate_request, async_req=True)
+        >>> result = thread.get()
+
+        :param async_req bool: execute request asynchronously
+        :param CreateEthereumEstimateRequest create_ethereum_estimate_request: (required)
+        :param _return_http_data_only: response data without head status code
+                                       and headers
+        :param _preload_content: if False, the urllib3.HTTPResponse object will
+                                 be returned without reading/decoding response
+                                 data. Default is True.
+        :param _request_timeout: timeout setting for this request. If one
+                                 number provided, it will be total request
+                                 timeout. It can also be a pair (tuple) of
+                                 (connection, read) timeouts.
+        :return: tuple(EstimateResponse, status_code(int), headers(HTTPHeaderDict))
+                 If the method is called asynchronously,
+                 returns the request thread.
+        """
+
+        local_var_params = locals()
+
+        all_params = ["create_ethereum_estimate_request"]  # noqa: E501
+        all_params.append("async_req")
+        all_params.append("_return_http_data_only")
+        all_params.append("_preload_content")
+        all_params.append("_request_timeout")
+        all_params.append("mass_g")
+        all_params.append("total_price_cents_usd")
+        all_params.append("project_id")
+        all_params.append("metadata")
+        all_params.append("distance_m")
+        all_params.append("transportation_method")
+        all_params.append("package_mass_g")
+        all_params.append("create_order")
+        all_params.append("make")
+        all_params.append("model")
+        all_params.append("year")
+        all_params.append("transaction_value_btc_sats")
+        all_params.append("timestamp")
+        all_params.append("gas_used")
```

Comment:
seems like this list of params is defined for every request in this file and consist of all possible params rather than the ones needed just for this particular endpoint. Not a blocker but do you happen to know the reason for this?

Reply:
Yes, this is a limitation of the current template / code generation. 

I'm adding a tracker here: https://app.asana.com/0/1200609773402056/1200634505135525/f